src/google/protobuf/util/field_mask_util.cc

```diff
@@ -52,17 +52,17 @@ std::string FieldMaskUtil::ToString(const FieldMask& mask) {
 void FieldMaskUtil::FromString(StringPiece str, FieldMask* out) {
   out->Clear();
   std::vector<std::string> paths = Split(str, ",");
-  for (int i = 0; i < paths.size(); ++i) {
-    if (paths[i].empty()) continue;
-    out->add_paths(paths[i]);
+  for (const std::string& path : paths) {
+    if (path.empty()) continue;
+    out->add_paths(path);
   }
 }
 
 bool FieldMaskUtil::SnakeCaseToCamelCase(StringPiece input,
                                          std::string* output) {
   output->clear();
   bool after_underscore = false;
-  for (int i = 0; i < input.size(); ++i) {
+  for (unsigned int i = 0; i < input.size(); ++i) {
```

Comment:
One more `unsigned` here, I think this one could be a range-based for loop.

Reply:
Done